pkgs/by-name/sl/slweb/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromSourcehut,
+  versionCheckHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "slweb";
+  version = "0.10.1";
+
+  src = fetchFromSourcehut {
+    owner = "~strahinja";
+    repo = "slweb";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-AJg8qgbNUKizU0uyTnq9EviIXOUuaGvQowLAyTWhGTY=";
+  };
+
+  postPatch = ''
+    substituteInPlace config.mk \
+      --replace-fail "/usr/local" "$out"
+  '';
+
+  buildPhase = ''
+    runHook preBuild
+
+    make slweb
```

Comment:
there is variable to do this. Was ist makeTarget? buildTarget? Something like that

Reply:
I tried both and unfortunately, it keeps trying to build examples before the application, which crashed.

Reply:
Actually the issue was `enableParallelBuilding` which has been introduced in this derivation when using the previous `redo` build system.

Now everything works normally.